docs/reference/dapis/chains/chains-list.md

```diff
@@ -10,15 +10,15 @@ DO NOT place line breaks between HTML element like <div>.
 They will not render as HTML but rather text or you get an error. No idea why!
 -->
 <div class="api3-bc-chains-box" v-for="(chain, index) in chains">
-    <div class="api3-bc-chains-name" :href="chain.explorerUrl+'/address/'+chain.contracts['Api3ServerV1']"
-      >{{ chain.fullName }} 
-    </div>
     <div class="api3-bc-chain-token">
       {{ chain.nativeToken }}
     </div>
     <div class="api3-bc-chains-id">
       Chain Id: <b>{{ chain.id }}</b>
     </div>
+    <div class="api3-bc-chains-name" :href="chain.explorerUrl+'/address/'+chain.contracts['Api3ServerV1']"
```

Comment:
Actually it looks like the `chain.explorerUrl` now ends with a `/` character, which, when added to `/address/`, creates a double `//`. Have a look by hovering over any of the explorer links on the preview build page: https://vitepress-docs--pr667-20231117-indexes-axi-32ysbsfe.web.app/reference/dapis/chains/

This results in the rootstock explorer URLs breaking e.g. 

New, not working: https://explorer.testnet.rsk.co//address/0x3dEC619dc529363767dEe9E71d8dD1A5bc270D76
Old, working: https://explorer.rsk.co/address/0x3dEC619dc529363767dEe9E71d8dD1A5bc270D76